corehq/apps/accounting/models.py

```diff
@@ -964,6 +1026,10 @@ def renew_subscription(self, date_end=None, note=None, web_user=None,
             date_start=self.date_end,
             date_end=date_end,
         )
+        if type is not None:
```

Comment:
`type` seems like a scary variable to use since it's a built-in python function. i'm assuming the python parameter takes precedence over the `type` python function, but we've now lost the use of that function. don't necessarily need to change, but when i first read this i thought it was a tautology